diff --git a/drivers/cpufreq/Kconfig b/drivers/cpufreq/Kconfig
--- a/drivers/cpufreq/Kconfig
+++ b/drivers/cpufreq/Kconfig
@@ -28,7 +28,13 @@
 
 config CPU_FREQ_BOOST_SW
 	bool
 	depends on THERMAL
+config INFINITY_CPU_PERF_TUNING
+	bool "Infinity Kernel CPU Performance Tuning"
+	depends on CPU_FREQ
+	default n
+	help
+	  CPU frequency boost and scheduler tuning for gaming.
 
 config CPU_FREQ_STAT
 	bool "CPU frequency transition statistics"
