diff --git a/mm/Kconfig b/mm/Kconfig
--- a/mm/Kconfig
+++ b/mm/Kconfig
@@ -248,7 +248,12 @@
 	  scenario aforementioned and helps improving memory defragmentation.
 
 #
 # support for memory compaction
+config INFINITY_BATTERY_OPT
+	bool "Infinity Kernel Battery Optimizations"
+	default y if INFINITY_KERNEL
+	help
+	  Battery life optimizations.
 config COMPACTION
 	bool "Allow for memory compaction"
 	def_bool y
