diff --git a/drivers/input/input.c b/drivers/input/input.c
--- a/drivers/input/input.c
+++ b/drivers/input/input.c
@@ -2569,4 +2569,9 @@
 }
 
 subsys_initcall(input_init);
+#ifdef CONFIG_INFINITY_CPU_PERF_TUNING
+static bool infinity_input_boost = true;
+module_param_named(input_boost, infinity_input_boost, bool, 0644);
+MODULE_PARM_DESC(input_boost, "Boost CPU on input events (default: true)");
+#endif
 module_exit(input_exit);
