diff --git a/drivers/block/zram/Kconfig b/drivers/block/zram/Kconfig
--- a/drivers/block/zram/Kconfig
+++ b/drivers/block/zram/Kconfig
@@ -1,4 +1,10 @@
 # SPDX-License-Identifier: GPL-2.0
+config INFINITY_ZRAM_LZ4
+	bool "Infinity Kernel: LZ4 compression for ZRAM"
+	depends on ZRAM && CRYPTO_LZ4
+	default y if INFINITY_KERNEL
+	help
+	  Use LZ4 instead of LZO for ZRAM compression.
 config ZRAM
 	tristate "Compressed RAM block device support"
 	depends on BLOCK && SYSFS && ZSMALLOC && CRYPTO
